frost/ciphersuite.go

```diff
@@ -46,6 +46,9 @@ type Curve interface {
 	// Identity returns elliptic curve identity element.
 	Identity() *Point
 
+	// Order returns the order of the elliptic curve.
+	Order() *big.Int
```

Comment:
For some potential future implementations, this comment will need to be amended to "returns the order of the group produced by the generator". On some curves the generator produces a prime-order subgroup, whose order is not the same as that of the entire curve.

Reply:
That's a really good point. Improved in e1c912ce48e7797a9e1470bca8cefd36b373c061.